scene/resources/3d/mesh_library.h

```diff
@@ -50,6 +50,7 @@ class MeshLibrary : public Resource {
 		String name;
 		Ref<Mesh> mesh;
 		Transform3D mesh_transform;
+		int mesh_cast_shadow = 1; // RenderingServer::ShadowCastingSetting::SHADOW_CASTING_SETTING_ON
```

Comment:
Shouldn't this use / duplicate the RenderingServer enum so it can be exposed and documented properly?

Reply:
I think the problem was back then some problem with the variant setters and getters but I may just add a big switch.